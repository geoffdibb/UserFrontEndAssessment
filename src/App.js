import React, { Component } from 'react';
import './App.css';


import Navbar from "./navbar";
export class App extends Component {

  render() {
    return (
      <div  style={{ display: 'flex', justifyContent: 'center', backgroundColor: "black", color: "white" }}>
        <Navbar/>

        
      </div>
    );
  }


}

export default App;