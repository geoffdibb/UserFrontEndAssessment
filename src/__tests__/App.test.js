import React from 'react';
import ReactDOM from 'react-dom';
import App from '../App';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

it('snapshot App', () => {
  // const wrapper = shallow(<App />);
  const tree = renderer.create(<App/>).toJSON()
  expect(tree).toMatchSnapshot();
})

it('renders without crashing', () => {
  const div = document.createElement('div');
  ReactDOM.render(<App />, div);
  ReactDOM.unmountComponentAtNode(div);
});
it('renders Navbar', () => {
  const wrapper = shallow(<App />);
      const Navbar = <Navbar/>;
  expect(wrapper.contains(Navbar)).toEqual(true);

});