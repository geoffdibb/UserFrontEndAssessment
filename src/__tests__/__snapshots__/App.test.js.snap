// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot App 1`] = `
<div
  style={
    Object {
      "backgroundColor": "black",
      "color": "white",
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <div>
    <h1
      backgroundcolor="black"
    >
      Home
    </h1>
    <button>
      <a
        href="/Home"
        onClick={[Function]}
      >
        Home
      </a>
    </button>
    <button>
      <a
        href="/Register"
        onClick={[Function]}
      >
        RegisterAccount
      </a>
    </button>
    <button>
      <a
        href="/Login"
        onClick={[Function]}
      >
        Login
      </a>
    </button>
    <button>
      <a
        href="/ViewUsers"
        onClick={[Function]}
      >
        View All Accounts
      </a>
    </button>
  </div>
</div>
`;
